### PR TITLE
Added "." on to check_dtype and obj parameter

### PR DESCRIPTION
**Pull request by Varun270**
- [] closes #xxxx
- [] tests added / passed
- [] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit) for how to run them
- [] whatsnew entry

From #28602
`pandas.testing.assert_categorical_equal: Parameter "check_dtype" description should finish with "."
pandas.testing.assert_categorical_equal: Parameter "obj" description should finish with "."`

### CODE REVIEW

**Comment by jreback** [comment]
can you merge master

File: pandas/core/frame.py [hunk 1]
```diff
@@ -10563,13 +10563,13 @@ def isin(self, values) -> DataFrame:
 
         When ``values`` is a Series or DataFrame the index and column must
         match. Note that 'falcon' does not match based on the number of legs
-        in df2.
+        in other.
 
-        >>> other = pd.DataFrame({'num_legs': [8, 2], 'num_wings': [0, 2]},
+        >>> other = pd.DataFrame({'num_legs': [8, 3], 'num_wings': [0, 2]},
```
Comment (phofl) [inline pandas/core/frame.py:10568]: Why did you change this?

Comment (Varun270) [reply pandas/core/frame.py:10568]: I didn't change these lines actually I only added the "." at the end of description of "check_dtype" and "obj". These changes are from a different issue that I solved and my PR got merged but I have no idea why these commits got pushed with this PR. You can have a look at this PR for better understanding - https://github.com/pandas-dev/pandas/pull/43820/files

Comment (Varun270) [reply pandas/core/frame.py:10568]: Do let me know what would I have to do in order to get rid of these extra commits.

Comment (phofl) [reply pandas/core/frame.py:10568]: Simply revert the relevant commit on your branch

Comment (MarcoGorelli) [reply pandas/core/frame.py:10568]: something like
```
git fetch upstream
git rebase upstream/master
git push origin HEAD -f
```

Comment (Varun270) [reply pandas/core/frame.py]: Done! Kindly review it again.

**Review by MarcoGorelli — APPROVED** [review]

**Comment by MarcoGorelli** [comment]
Thanks @Varun270 - next time I'd advise to make a new branch for each issue you try to solve

**Comment by Varun270** [comment]
> Thanks @Varun270 - next time I'd advise to make a new branch for each issue you try to solve

Noted, I Will keep it in mind!
